test/lock-parser/fixtures/multiple-remotes/out.json

```diff
@@ -0,0 +1,1451 @@
+{
+  "dependencies": [
+    {
+      "name": "Antlr",
+      "options": {},
+      "version": "3.5.0.2",
+      "remote": "http",
```

Comment:
remote url is missing

Reply:
Yes, due to incorrect splitting of remote url attribute. Fixing this in https://github.com/snyk/snyk-paket-parser/pull/16/files#r255176589 and then regenerating fixtures accordingly.